src/sentry/incidents/endpoints/organization_alert_rule_index.py

```diff
@@ -33,15 +45,72 @@ def post(self, request, organization):
         """
         Create an alert rule
         """
+        # TODO: Can save all created objects in a list for a rollback in case there are any errors. Just pass list to rollback function.
+        # TODO: Use create_alert_rule, create_alert_rule_trigger, and create_alert_rule_trigger_action?
         if not features.has("organizations:incidents", organization, actor=request.user):
             raise ResourceDoesNotExist
 
-        serializer = AlertRuleSerializer(
-            context={"organization": organization, "access": request.access}, data=request.data
+        data = deepcopy(request.data)
+        # TODO: Possibly before creating the rule, or sometime - ensure at least one trigger is defined. Also ensure that each trigger has at least one action.
```

Comment:
It might make sense to have an overall `Serializer` that handles the creation of the entire rule, and enforces the minimum one trigger, and the threshold constraints, etc. It could make sense to just build this all into the `AlertRuleSerializer`, potentially? We could still use the other serializers to verify that specific data.